dask/dataframe/tests/test_dataframe.py

```diff
@@ -963,16 +963,18 @@ def test_append2():
     dsk = {('x', 0): pd.DataFrame({'a': [1, 2, 3], 'b': [4, 5, 6]}),
            ('x', 1): pd.DataFrame({'a': [4, 5, 6], 'b': [3, 2, 1]}),
            ('x', 2): pd.DataFrame({'a': [7, 8, 9], 'b': [0, 0, 0]})}
-    ddf1 = dd.DataFrame(dsk, 'x', ['a', 'b'], [None, None])
+    meta = make_empty_frame({'a': 'i8', 'b': 'i8'})
+    ddf1 = dd.DataFrame(dsk, 'x', meta, [None, None])
```

Comment:
In some cases it might make sense to start creating dataframe objects with `dd.from_pandas` instead of manually by hand.  This style of test is mostly a holdover from before `from_pandas`.